practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -109,12 +112,11 @@ public PractitionerDetails getPractitionerDetails(
             IBaseResource practitioner =
                     practitioners.size() > 0 ? practitioners.get(0) : new Practitioner();
             Long practitionerId = null;
-            String practitionerIdStringValue = null;
+            String practitionerIdStringValue;
```

Comment:
We can set the defaults to `0L` and `""` respectively to coincide with the recommended refactors below